.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
         with:
-          path: ansible_collections/middleware_automation/redhat_csp_download
+          path: ansible_collections/redhat/redhat_csp_download
```

Comment:
@sabre1041 @hcherukuri I think this kind of changes will be Hell to deal when we need to setup the upstream of this. I would suggest we look at way to have a variable with the namespace, if this file can support it.

Reply:
Yes GitHub can support setting variables and we can set the appropriate variable based on the branch being checked out

Reply:
Yes we can add something like this, if [ "$CI_COMMIT_REF_NAME" = "main" ]; then namespace="redhat"; else namespace="middleware_automation"; fi

Reply:
something similar to that, yup! GH Actions has its own set of internal variables that you can use

Reply:
take a look into updated changes.